config/drupal-10/drupal-10.1-deprecations.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+
+use DrupalRector\Rector\Deprecation\FunctionToStaticRector;
+use DrupalRector\Rector\ValueObject\FunctionToStaticConfiguration;
+use Rector\Config\RectorConfig;
+use Rector\PHPUnit\Set\PHPUnitLevelSetList;
+use Rector\Symfony\Set\SymfonyLevelSetList;
+use Rector\Symfony\Set\TwigLevelSetList;
+
+return static function (RectorConfig $rectorConfig): void {
+    $rectorConfig->sets([
+        PHPUnitLevelSetList::UP_TO_PHPUNIT_90,
+        SymfonyLevelSetList::UP_TO_SYMFONY_63,
+        TwigLevelSetList::UP_TO_TWIG_240,
+    ]);
```

Comment:
Do we need these here if in `config/drupal-10/drupal-10.0-deprecations.php`?

Reply:
You are right, its not consistent with the other rulesets. Just need the symfony one since that minor got upped.